packages/rollup-plugin/test/rollupUtils.test.ts

```diff
@@ -7,33 +7,77 @@ import * as td from 'testdouble'
 
 describe('extractSourcemapDataFromBundle', () => {
   const outputOptions = td.object<NormalizedOutputOptions>()
-  outputOptions.dir = 'dist' 
+  outputOptions.dir = 'dist'
 
   it('should return an array with sourcemap file data', () => {
-    const data = extractSourcemapDataFromBundle(outputOptions, bundle)
+    const data = extractSourcemapDataFromBundle(outputOptions, bundle, [])
     expect(data).to.be.an('array').lengthOf(3)
     expect(data).to.have.deep.members([
       {
-        sourcemapFilename: 'index.js.map', 
-        sourcemapFilePath: path.resolve('dist/index.js.map'), 
-        jsFilename: 'index.js', 
+        sourcemapFilename: 'index.js.map',
+        sourcemapFilePath: path.resolve('dist/index.js.map'),
+        jsFilename: 'index.js',
         jsFilePath: path.resolve('dist/index.js')
-      }, 
+      },
       {
-        sourcemapFilename: 'foo.js.map', 
-        sourcemapFilePath: path.resolve('dist/foo.js.map'), 
-        jsFilename: 'foo.js', 
+        sourcemapFilename: 'foo.js.map',
+        sourcemapFilePath: path.resolve('dist/foo.js.map'),
+        jsFilename: 'foo.js',
         jsFilePath: path.resolve('dist/foo.js')
       },
       {
-        sourcemapFilename: 'subfolder/bar.js.map', 
-        sourcemapFilePath: path.resolve('dist/subfolder/bar.js.map'), 
-        jsFilename: 'subfolder/bar.js', 
+        sourcemapFilename: 'subfolder/bar.js.map',
+        sourcemapFilePath: path.resolve('dist/subfolder/bar.js.map'),
+        jsFilename: 'subfolder/bar.js',
         jsFilePath: path.resolve('dist/subfolder/bar.js')
       },
     ])
   })
-});
+
+  const itEach = ['foo.js', '**/foo.js', 'foo.*', '**/foo.*', 'foo.js*', '**/foo.js*']
```

Comment:
Your readme change and TS types say that `ignorePaths` accepts regex, but I'm not sure if it actually does? If it does accept regex, you should add one to these tests.

Reply:
The regex part of the readme is not there anymore, but I forgot to remove the TS types! On it!